src/platform/packages/shared/kbn-unified-tabs/src/components/tabs_bar_menu/tabs_bar_menu.tsx

```diff
@@ -103,6 +103,7 @@ export const TabsBarMenu: React.FC<TabsBarMenuProps> = React.memo(
         panelPaddingSize="none"
         anchorPosition="downRight"
         hasArrow={false}
+        buffer={0}
```

Comment:
This fixes the dropdown issue when using the correct padding, so it remains below the button instead of getting pushed to the side.